### DISCUSSION TITLE
148

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#87] [General]
# 潮流周刊第148期 - 开车回家

封面图来源于国庆湖南回杭州途径景德镇服务区所拍的一张瓷器图，不得不说景德镇的瓷器的审美还是不错的。此外由于国庆未发潮流周刊，本期周刊加量不加价，哈哈哈。

https://weekly.tw93.fun/posts/148-%E5%BC%80%E8%BD%A6%E5%9B%9E%E5%AE%B6

**Comment by PTXXXXXZ** [comment]
老哥探索能力太强了！

**Comment by Nicol-Zhang** [comment]
最近换了mac在这里收藏的好物终于可以用上了

**Comment by 14790897** [comment]
强，第一次看，有很多实用的工具
